src/components/TodoView.js

```diff
@@ -0,0 +1,102 @@
+import PropTypes from "prop-types";
+import React from "react";
+import AddTodoForm from "./AddTodoForm";
+import TodoList from "./TodoList";
+import styles from "./SharedStyles.module.css";
+import { FaSortAlphaDown, FaSortAlphaUpAlt } from "react-icons/fa";
+import Toggle from "./Toggle";
+import Footer from "./Footer";
+import GoBack from "./GoBack";
+
+const TodoView = ({
+  onAddTodo,
+  fetchData,
+  updateData,
+  removeTodo,
+  isLoading,
+  todoList,
+  sortOrder,
+  setSortOrder,
+  sortData,
+  isDarkMode,
+  handleSwitch,
+}) => {
+  return (
+    <>
+      {isLoading ? (
```

Comment:
Nitpick:
This is a bit hard to read. Instead, you can use an early return to avoid having to use a ternary operator within React fragments:
```js
const TodoView = ({ ... }) => {
  if (isLoading) return <p>Loading...</p>;
  
  return (
    <div>...</div>
  );
}